content/jobs/healers/sage/_index.md

```diff
@@ -1,5 +1,6 @@
 ---
 authors:
   - Balance SGE Staff
+job_name: sage
```

Comment:
Missing title for sage

Reply:
This is actually okay, since this relies on us updating titles for the editor config (see issue #635), which is the next step after we merge this PR.